Fuzzing/rstp.txt

```diff
@@ -0,0 +1,354 @@
+/
+/0
+/0/video1
+/1
+/1.AMP
+/1/1:1/main
+/1/cif
+/1/stream1
+/11
+/12
+/4
+/CAM_ID.password.mp2
+/CH001.sdp
+/GetData.cgi
+/H264
+/HighResolutionVideo
+/HighResolutionvideo
+/Image.jpg
```

Comment:
This one is a duplicate of another line. Please make sure to `sort -u` this file so duplicates are removed.